packages/integration-tests/projects/suite-web/plugins/index.js

```diff
@@ -218,5 +260,38 @@ module.exports = on => {
             controller.disconnect();
             return null;
         },
+        resetCRI: async () => {
+            if (client) {
+                await client.close();
+                client = null;
+            }
+
+            return Promise.resolve(true);
+        },
+        activatePrintMediaQuery: async () => {
+            client = client || (await CDP({ port }));
+
+            return client.send('Emulation.setEmulatedMedia', { media: 'print' });
+        },
```

Comment:
I think that we will never test/implement it as probably no one will print Suite on the paper 😆

Reply:
what about transaction export and printig that ? 😆

Reply:
probably not :wink: haha.

Reply:
not really needed, I removed that.